src/api-requests/get-industry-events.ts

```diff
@@ -43,6 +43,33 @@ export const industryEventsFields = `
   location
   slug
   start_time
+  registration_url
```

Comment:
github won't let me add a comment earlier in the file, but I think this query is missing the `title` field under `authors`. i noticed that the authors' titles were missing on my local. looks like the code is there in the JSX to display it, it just needs to be grabbed from here.